src/test/java/plugins/AntPluginTest.java

```diff
@@ -68,10 +137,69 @@ public void locallyInstalledAnt() {
         step.targets.set("-version");
         job.save();
 
-        String expectedVersion = localAntVersion(antHome);
+        String expectedVersion = "1.10.5"; // this is the version installed in the java container by the ubuntu bionic
         job.startBuild().shouldSucceed().shouldContainsConsoleOutput(Pattern.quote(expectedVersion));
     }
 
+    @Test
+    @WithPlugins({"workflow-job", "workflow-cps", "workflow-basic-steps", "workflow-durable-task-step"})
+    public void testAntWrapper() {
+        useSlave();
+        String antHome = setUpAntInstallation();
+
+        String expectedVersion = "1.10.5"; // this is the version installed in the java container by the ubuntu bionic;
+
+        WorkflowJob workflowJob = jenkins.jobs.create(WorkflowJob.class);
+        workflowJob.script.set(SCRIPT_PIPELINE_ANT);
```

Comment:
you will need to tell the pipeline to use the ssh agent you have created - otherwise it will run anywhere.

this is inline in the pipeline definition `node(_the_agent_label) {`  or you would need to remove the ability to execute on the controller by setting the number of executors there to zero.

As the pipeline definition is not re-used in any other test it it would probably make sense to move the definition of it inline here.